# Request
bool req
---
bool joystick
bool autonomous
